src/e2e/java/teammates/e2e/cases/e2e/StudentHomePageE2ETest.java

```diff
@@ -0,0 +1,141 @@
+
+        logout();
+        studentHome = getHomePageNew().clickStudentLogin().loginAsStudent(unregUserId, unregPassword);
+
+        browser.waitForPageLoad();
+        assertTrue(studentHome.verifyStrangerMessage());
+
+        ______TS("login successfully");
+
+        logout();
+        studentHome = getHomePageNew().clickStudentLogin()
+                                   .loginAsStudent(TestProperties.TEST_STUDENT1_ACCOUNT,
+                                                   TestProperties.TEST_STUDENT1_PASSWORD);
+
+        browser.waitForPageLoad();
+        assertTrue(studentHome.verifyFirstPanelFeedbackSessionName("[SHomeUiT.CS1101]: Programming Methodology"));
+
+        ______TS("login successfully as admin (masquerade mode)");
+
+        AppUrl detailsPageUrl = createUrl(Const.WebPageURIs.STUDENT_HOME_PAGE)
+                .withUserId(testData.students.get("SHomeUiT.charlie.d@SHomeUiT.CS2104").googleId);
+
+        StudentHomePageNew studentHomePage = loginAdminToPageNew(detailsPageUrl, StudentHomePageNew.class);
+        browser.waitForPageLoad();
+
+        assertTrue(studentHomePage.verifyAdminPagePresent());
+    }
+
+    private void testLinks() {
+
+        AppUrl homePageUrl = createUrl(Const.WebPageURIs.STUDENT_HOME_PAGE)
+                .withUserId(testData.students.get("SHomeUiT.charlie.d@SHomeUiT.CS2104").googleId);
+
+        StudentHomePageNew studentHomePage = loginAdminToPageNew(homePageUrl, StudentHomePageNew.class);
+
+        ______TS("link: view team link");
+
+        studentHomePage.clickViewTeam();
+
+        ______TS("link: help page");
+
+        studentHomePage.loadStudentHelpTab();
+
+        ______TS("link: student profile page");
+
+        studentHomePage.loadStudentProfileTab();
+
+        ______TS("link: student home page");
+
+        studentHomePage.loadStudentHomeTab();
+
+        ______TS("link: log out to home page");
+
+        studentHomePage.clickLogout();
+    }
+
+    private void testLinkAndContentAfterDelete() throws Exception {
+
+        AppUrl detailsPageUrl = createUrl(Const.WebPageURIs.STUDENT_HOME_PAGE)
+                             .withUserId(testData.students.get("SHomeUiT.charlie.d@SHomeUiT.CS2104").googleId);
+
+        StudentHomePageNew studentHomePageNew = loginAdminToPageNew(detailsPageUrl, StudentHomePageNew.class);
+
+        ______TS("access the feedback session exactly after it is deleted");
+
+        BackDoor.deleteFeedbackSession("First Feedback Session", "SHomeUiT.CS2104");
+        studentHomePageNew.clickSubmitFeedbackButton("First Feedback Session");
+        studentHomePageNew.waitForPageToLoad();
+
+    }
+
+    @AfterClass
+    public void classTearDown() {
+        // The courses of test student1 account is not the same as `StudentProfilePageUiTest`
+        // so it has to be cleared before running that test. This is the reason why `StudentProfilePageUiTest`
+        // would fail if we don't remove the data bundle in this test.
```

Comment:
This is no longer (or not yet) the case now, so the block has no reason to be here (yet). Let's remove it.